### PR TITLE
fix/501

### LINKED ISSUE #501: python -m scanapi doesn't work as intended
## Bug report
ScanAPI includes a `__main__.py` file suggesting that it can be run as a module via `python -m scanapi`, but it doesn't seem to work properly:
```
$ python -m scanapi
/<some_path>/lib/python3.7/runpy.py:125: RuntimeWarning: 'scanapi.__main__' found in sys.modules after import of package 'scanapi', but prior to execution of 'scanapi.__main__'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
```

### Environment
- Operating System: macOS
- Python version: 3.7

### Expected behavior ?
I'm not sure about the original intended behaviour, but maybe it should just run the same CLI as the entry point?

I think an advantage of this would be that one could debug the CLI via the PDB debugger like this:
```bash
$ python -m pdb -m scanapi
```

### How to reproduce the the bug ?
Activate the virtual environment you installed ScanAPI in, then run `python -m scanapi`.

### Anthing else we need to know?
If running the same CLI as the entrypoint is the entended behaviour of `python -m scanapi`, then it can be achieved with the following changes:
- Rename `__main__.py` to `cli.py` (or something else?)
- Add a new `__main__.py` with
```python
from scanapi.cli import main

main()
```
- delete everything from `__init__.py`?

### PR DESCRIPTION
**Pull request by uesleicarvalhoo**
## Motivation behind this PR?
Issue #501 

## What type of change is this?
Bug fix

## Checklist

- [X]  I have added a changelog entry / my PR does not need a new changelog entry. [Instructions](https://github.com/scanapi/scanapi/wiki/Changelog).
- [X] I have added/updated unit tests. [Instructions](https://github.com/scanapi/scanapi/wiki/Writing-Tests).
- [X] New and existing unit tests pass locally with my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally#tests)
- [X] I have self-documented code my changes by adding docstring(s) and comment(s). [Instructions](https://github.com/scanapi/scanapi/wiki/First-Pull-Request#7-make-your-changes)
- [X] Current PR does not significantly decrease the code coverage and docstring coverage.
- [X] My code follows the style guidelines of this project.
- [X] I have run ScanAPI locally and manually tested my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally).
- [X] I have squashed my commits. [Instructions](https://github.com/scanapi/scanapi/wiki/Squashing-Commits).

## Issue
Closes #501

### CODE REVIEW

**Review by Crocmagnon — APPROVED** [review]
Thanks! LGTM :shipit:

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- Move content of `scanapi/__main__.py` to `scanapi/cli.py` to enable debug with `pdb` [#501](https://github.com/scanapi/scanapi/issues/501)
```
Comment (camilamaia) [inline CHANGELOG.md:11]:
```suggestion
- python -m scanapi command [#501](https://github.com/scanapi/scanapi/issues/501)
```

File: scanapi/__main__.py [hunk 2]
```diff
@@ -1,110 +1,3 @@
-import logging
+from .cli import main
```
Comment (camilamaia) [inline scanapi/__main__.py:1]:
```suggestion
from scanapi..cli import main
```

Just a tiny suggestion to use an absolute import, as you did for the others, in order to follow PEP8 [recommendation](https://peps.python.org/pep-0008/#imports)

**Review by camilamaia — COMMENTED** [review]
@uesleicarvalhoo Thanks for this fix 🙇‍♀️ I left just 2 minor suggestions :)

File: scanapi/__main__.py (hunk 2 above)
Comment (Crocmagnon) [reply scanapi/__main__.py:1]: Isn't there an extra dot here?

Comment (camilamaia) [reply scanapi/__main__.py:1]: Ohhhhh 🤣 🤦‍♀️ 

Sure, let me fix it. Great catch, thanks

Comment (Crocmagnon) [reply scanapi/__main__.py:1]: ☺️

Comment (uesleicarvalhoo) [reply scanapi/__main__.py:1]: sure! updated :)

**Pull request merged**